modules/standard/Sort.chpl

```diff
@@ -274,8 +274,14 @@ const defaultComparator: DefaultComparator = new DefaultComparator();
    default sort order.
 
  */
-const reverseComparator: ReverseComparator(DefaultComparator);
-reverseComparator = new ReverseComparator();
+@deprecated("The variable 'reverseComparator' is now deprecated, please create a new instance of the :type:`ReverseComparator` type reversing the :type:`DefaultComparator` instead.")
```

Comment:
Consider "please create a new instance of the :type:`reverseDefaultComparator`"

Reply:
I originally had that, but since you can't use the type alias with `new`, I figured users would run into less problems if we directed them to use the general type.  But I can also undo that pretty easily